### DISCUSSION TITLE
additionalcountries setting

### DISCUSSION THREAD

**Discussion by nicward** [microsoft/AL-Go#479] [Q&A]
Hello,
I'm just starting to setup AL-go to manage our BC developments.  Could someone advise on how use the additionalcountries setting.  I tried to add this but I'm just not getting it right.  I'm not sure if I'm adding it in the wrong setting file or if I have the format wrong.

Thanks

Nick

**Comment by nicward** [comment]
This has been resolved now, I just needed to get the format right.

**Comment by freddydk** [comment]
In case other people run into the same, format is a json array: [ "us", "dk" ]

**Comment by nicward** [comment]
I made a very silly mistake, I was using "uk" instead of "gb".

Sorry!

**Comment by freddydk** [comment]
No problem
